sql/moz-fx-data-shared-prod/mozilla_vpn_derived/login_flows_v1/query.sql

```diff
@@ -13,11 +13,38 @@ WITH base AS (
       ) IGNORE NULLS
     ) AS fxa_uids,
     LOGICAL_OR(event_type = "fxa_email_first - view") AS viewed_email_first_page,
+    ARRAY_AGG(
+      IF(
+        entrypoint_experiment IS NOT NULL
+        OR entrypoint_variation IS NOT NULL
+        OR utm_campaign IS NOT NULL
+        OR utm_content IS NOT NULL
+        OR utm_medium IS NOT NULL
+        OR utm_source IS NOT NULL
+        OR utm_term IS NOT NULL,
+        STRUCT(
+          entrypoint_experiment,
+          entrypoint_variation,
+          CAST(NULL AS STRING) AS referrer,
```

Comment:
blocking this on figuring out referrer with @birdsarah

Reply:
I don't think fxa_email_first - view is the right event any more. Users will land on a subscription platform page. Not the accounts login page.

Reply:
> I don't think fxa_email_first - view is the right event any more. Users will land on a subscription platform page. Not the accounts login page.

Okay, I'll rework the use of that in the funnel as a separate issue.